libcontainer/configs/cgroup_deprecated.go

```diff
@@ -0,0 +1,29 @@
+package configs
+
+import cg "github.com/opencontainers/runc/libcontainer/cgroups/configs"
+
+// Deprecated: use [github.com/opencontainers/runc/libcontainer/cgroups/configs].
```

Comment:
Not really important (just an alternative) as these don't point to specific types, I think it's possible to just  add the `// Deprecated:` to the package;


```go
package configs // Deprecated: use [github.com/opencontainers/runc/libcontainer/cgroups/configs].
```

Reply:
done